.github/workflows/coverall.yml

```diff
@@ -34,7 +34,6 @@ jobs:
               with:
                   github-token: ${{ secrets.GITHUB_TOKEN }}
                   flag-name: run-${{ matrix.tests }}
-                  path-to-lcov: ./coverage/${{ matrix.tests }}/lcov.info
```

Comment:
comment: not so experienced with coveralls, can you help me understand why we should remove this?

Reply:
Yes, sure. We are using this GitHub action and that input is not part of the list: https://github.com/coverallsapp/github-action?tab=readme-ov-file#inputs

Reply:
Okay then! Tysm!